R/DBConnection.R

```diff
@@ -106,6 +106,39 @@ setGeneric("dbSendQuery",
   valueClass = "DBIResult"
 )
 
+#' @export
+dbBreak <- structure(list(), class = "dbi_break_sentinel")
+
+dbFetchChunked <- function(rs, n, callback) {
```

Comment:
(BTW I think `dbFetchChunked` was supposed to be named `dbFetchChunkedQuery` and exported.)

I see what you mean, but I like the `dbGetQuery => dbGetChunkedQuery`, `dbFetchQuery => dbFetchChunkedQuery` parallel. Think of the "chunked" modifier as saying instead of getting a result back, you receive the results incrementally through a callback. (Maybe "chunked" should be suffix instead of in the middle, i.e. `dbGetQueryChunked`.) This "chunked" stuff is a general pattern we should make greater use of in the R community, IMHO. `read_csv`, `httr::GET`, for examples.

In other words, don't think of it as changing from "fetching" to "processing"--you're still fetching, just receiving the results in a  different way.